packaging/config/agent.yaml

```diff
@@ -0,0 +1,29 @@
+# Path where Consul auto-loads configuration files: the node meta-data will be stored there as `trento-metadata.json`
```

Comment:
Just nitpicking, but I'd add an intro in the comments describing the purpose of the file, e.g. -> "This is the configuration for trento when executed under the `agent` role ..." and describe also a bit the order of precedence. I know that this stuff is probably already described elsewhere in the docs, but it's IMO a good thing to have it here too.

Feel free to ignore this for now as we can adress it in a future PR.

Reply:
Makes sense, I added some :wink: